### ISSUE TITLE
When teleporting across dimensions, it doesn't remount you.

### ISSUE THREAD

**Issue by da3dsoul** [planetguy32/RemainInMotion#146]

**Comment by da3dsoul** [comment]
If you are riding something.

**Comment by TorakWolf** [comment]
Pretty much aesthetic, but not gonna lie, mounting your personal Bibliocraft Armchair after teleporting your battleship to the front line makes you fell frickin badass. Or that rare Pegasus you took years to breed.

**Comment by da3dsoul** [comment]
Exactly! And who doesn't like cross dimensional rollercoasters. I've been looking at the way it saves the mount on world close. You spawn mounted in a minecart if you quit in one.
